public/src/main/java/com/revenuecat/purchases/VerificationResult.kt

```diff
@@ -21,7 +21,7 @@ enum class VerificationResult {
     /**
      * Entitlements were verified with our server.
```

Comment:
I wouldn't mention entitlements here since this type can be used for any response.

Reply:
Hmm good point... We should also update the other enum values and iOS @NachoSoto